app/src/Homepage/Homepage-components.tsx

```diff
@@ -22,10 +43,19 @@ export const Sidebar: React.FC<SidebarProps> = ( props: SidebarProps ) => {
 
 
 export const LevelTab: React.FC<{ classlevel: string }> = ({ classlevel }) => {   
+    const navigate = useNavigate();
     
     const handleClick = () => {
-        console.log('Button clicked: ' + classlevel);
-        
+        if (classlevel === 'Home') {
+            // If 'Home' button, route to default homepage
+            navigate('/rate-my-cse/');
```

Comment:
Can you make this a variable and then reference it? That way, if we need to change the route from /rate-my-cse/, we don't have to look for each location

Reply:
It may be best to use some kind of global variable/function that calls on this variable that other files can reference as well since its used in quite a bit of places

Reply:
I'm leaning towards a global function that returns a string. That way you can add a unit test ensuring that function always returns that string

Reply:
Replaced all uses with global function, made new utils file for it
Lmk if you wanted that implemented in a different way